keras_cv/models/object_detection/__internal__.py

```diff
@@ -0,0 +1,130 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import functools
+import math
+
+import numpy as np
+import pandas as pd
```

Comment:
Rather than requiring `pandas` (which is heavy) just to support the optional use case of passing a dataframe, I recommend making it an optional dependency, like https://github.com/keras-team/keras/blob/master/keras/engine/data_adapter.py#L42-L45